btr-web/btr-main-app/components/individual-person/SummaryTable.vue

```diff
@@ -105,6 +111,21 @@
           </td>
         </template>
       </tr>
+      <!-- standard class or css style without !important was not working -->
+      <tr
+        v-if="displayAdditional(item) && editingIndex != index"
+        style="border-top-width: 0!important"
+        data-cy="summary-table-external-influence"
+      >
+        <td colspan="6">
+          <p v-if="item.externalInfluence === ExternalInfluenceE.CAN_BE_INFLUENCED">
```

Comment:
The text you're showing here doesn't match the design ticket fyi

Reply:
This was previously  also a text of radio buttons, the thing is the text changed, but I think summary table one is not updated.
Will need to check with Scott on that one, so lets UX doublecheck that one.

Reply:
Yeah, Scott confirmed.